src/components/SingleName/NameRegister/PriceCalculator.js

```diff
@@ -6,16 +6,22 @@ function priceCalculator({
 }) {
   const priceInEth = new EthVal(`${price}`).toEth()
   const premiumInEth = new EthVal(`${premium}`).toEth()
+  let priceInUsd, premiumInUsd, basePriceInUsd
+  if (ethUsdPrice) {
```

Comment:
The code inside the if looks a bit odd, why is there a ';' at the start of line 11, and why is each line is wrapped in '()' ? Looks like all the variable assignments are being done in one expression but it can be split up.